hypervisor/qemu/qemu.go

```diff
@@ -25,22 +26,27 @@ import (
 )
 
 type VMConfig struct {
+	// general
```

Comment:
Please, don't add these comments. I know you want well, but it hurts my eyes how bad this gets formatted in the end: https://github.com/miha-plesko/capstan/blob/5f3c49884ba2ba5351bff467b4b551801a9bb904/hypervisor/qemu/qemu.go#L28-L49

Reply:
If necessary, add the comment right to first attribute that starts a new attribute group.